build/storage/scripts/disk_infrastructure.py

```diff
@@ -25,33 +25,27 @@ def get_number_of_virtio_blk(sock: str) -> int:
     return number_of_virtio_blk_devices
 
 
-def is_virtio_blk_attached(sock: str) -> int:
+def is_virtio_blk_attached(sock: str) -> bool:
```

Comment:
you inverted the logic returning False instead of 1 and True instead of 0. I assume this is right now (according to function naming)

Reply:
This function should return True if a virtio_blk is attached and it does return True in that case. It returned 0 before because that was a return code for bash function. Therefore logic seems to be inverted because True is 1 and False is 0, but bash needs a return code so I need to return 0 in case of True and 1 in case of False.